docs/Tutorial_Consensus_Motif.ipynb

```diff
@@ -0,0 +1,610 @@
+{
```

Comment:
<p>It would be helpful to mention that you are using scipy for clustering here. We want to avoid any confusion with readers thinking that time series clustering is offered in STUMPY.</p><p> </p><p>Additionally, I think it would be nice to state upfront what you'd expect a "successful" clustering of the organisms would be so that it is more obvious when the clustering is bad (below). Like even just one sentence may be sufficient</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/279/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>